js/binance.js

```diff
@@ -134,8 +134,6 @@ module.exports = class binance extends Exchange {
                     'dapiPublic': 'https://testnet.binancefuture.com/dapi/v1',
                     'dapiPrivate': 'https://testnet.binancefuture.com/dapi/v1',
                     'dapiPrivateV2': 'https://testnet.binancefuture.com/dapi/v2',
-                    'eapiPublic': 'https://testnet.binanceops.com/eapi/v1',
```

Comment:
@Dan-krm, this by itself is not a fix because our fetchMarkets will try to use these endpoints anyway, so it still fails (with a different error). I will try to patch it properly